modules/ROOT/pages/error-handling.adoc

```diff
@@ -48,7 +48,7 @@ a reference:
 [source,xml,linenums]
 ----
 <on-error-continue name="loggingErrorHandler">
-    <logger message="#['Error: ' ++ error.description ++ ', Payload: ' ++ payload]"/>
+    <logger message="#['Error: ' ++ error.description]"/>
```

Comment:
removing  the payload part bc it can produce an error if the input depending on the type of the payload (see https://www.mulesoft.org/jira/browse/DOCS-3940)